docs/source/gs-processing/gs-processing-getting-started.rst

```diff
@@ -1,5 +1,7 @@
+.. _gs-processing:
+
 GraphStorm Processing Getting Started
-=====================================
+=======================================
```

Comment:
Generally let's try to have the heading indicators match the number of characters of the section title.

Reply:
Sure. Changed.